### ISSUE TITLE
Typo in Waiting on Data section: instruction on spin package has incorrect sachag as author.

### ISSUE THREAD

**Issue by rts100** [DiscoverMeteor/book#324]
Book states "Meteor add: sachag:spin"

Atmosphere shows package is sacha:spin.

Great book, great publishing format.  Love the updates.

Thanks,

Richard

**Comment by rts100** [comment]
Sorry, duplicate of issue 317.
